### PR TITLE
Feature-policy-parallel-dev

### PR DESCRIPTION
**Pull request by abeowlu**
extracted policy implementation user interface with arax.
Conditional bucket notification implementation for data ingest.

### CODE REVIEW

File: .github/workflows/aws-sqs-moduleTest.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+#   Documentation:
+#     - https://www.terraform.io/docs/cloud/users-teams-organizations/api-tokens.html
+#     - https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
+#
+# 3. Reference the GitHub secret in step using the `hashicorp/setup-terraform` GitHub Action.
+#   Example:
+#     - name: Setup Terraform
+#       uses: hashicorp/setup-terraform@v1
+#       with:
+#         cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
```
Comment (charlesa101) [inline .github/workflows/aws-sqs-moduleTest.yml]: remove this if its not all needed please

File: .github/workflows/aws-sqs-moduleTest.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          export TF_VAR_secret_key: ${{ secrets.MAVEN_AWS_SECRET_KEY }}
+
+      ## Using terraform implementation in workflow above. Can be changed to AWS configuration below.
+
+      #     - name: Configure AWS Credentials
+      #       uses: aws-actions/configure-aws-credentials@v1
+      #       with:
+      #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      #         aws-region: us-west-1
```
Comment (charlesa101) [inline .github/workflows/aws-sqs-moduleTest.yml]: not sure why we need this here

Comment (abeowlu) [reply .github/workflows/aws-sqs-moduleTest.yml]: removed.

File: .github/workflows/aws-sqs-moduleTest.yml [hunk 3]
```diff
@@ -2,63 +2,28 @@
...
+      - master
     paths-ignore:
-      -'docs/**'
-      -'Docs/**'
+      - "docs/**"
+      - "Docs/**"
   pull_request:
+    branch:
+      - main
+      - master
```
Comment (charlesa101) [inline .github/workflows/aws-sqs-moduleTest.yml:19]: not sure if we need the `master` branch any more

File: .github/workflows/aws-sqs-moduleTest.yml [hunk 4]
```diff
@@ -68,56 +33,44 @@ jobs:
...
-# #       with:
-# #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-# #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-# #         aws-region: us-west-1
+      #     - name: Configure AWS Credentials
+      #       uses: aws-actions/configure-aws-credentials@v1
+      #       with:
+      #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      #         aws-region: us-west-1
```
Comment (charlesa101) [inline .github/workflows/aws-sqs-moduleTest.yml:58]: if we are not using this can you clean it up?

Comment (abeowlu) [reply .github/workflows/aws-sqs-moduleTest.yml:58]: Cleaned up.

**Review by charlesa101 — APPROVED** [review]
LGTM

**Pull request merged**